src/mem/ruby/network/MessageBuffer.hh

```diff
@@ -124,6 +125,7 @@ class MessageBuffer : public SimObject
     const MsgPtr &peekMsgPtr() const { return m_prio_heap.front(); }
 
     void enqueue(MsgPtr message, Tick curTime, Tick delta,
+                bool ruby_is_random, bool ruby_warmup,
```

Comment:
Can we have some documentation on what there two new parameters mean?

Reply:
Does #1735 suffice?  These are basically the former `RubySystem::getRandomization()` and `RubySystem::getWarmupEnabled()`